cpp/data/email_watcher.conf

```diff
@@ -0,0 +1,26 @@
+# Each section must provide the following entries:
+# from_host                    -- where the emails are originating
+# subject_pattern              -- a PCRE regex that must match the emails' subjects
+# email_body_positive_pattern  -- a PCRE that must match the emails' bodies
+# email_body_negative_pattern  -- a PCRE that must not match the emails' bodies
+#                                 At least one of email_body_positive_pattern and
+#                                 email_body_positive_pattern must be specified in
+#                                 each section!
+# overdue_time_window          -- a time interval in hours specifying after what time
+#                                 we should get worried if we have not yet received an email
+#                                 matching the other criteria
+
+# The following is a comma-separated list of email address
+notify=johannes.ruscheinski@uni-tuebingen.de
+
+# Where to move the mbox files after processing:
+backup_dir_path=/tmp
+
+# Forward all emails with a priority like this or higher
+forward_priority = 5
```

Comment:
maybe 4 would be better (anything higher than 3=normal)

Reply:
This is not our actual config file but just one I use for testing.  I only committed it because I wanted you to have it for documentation purposes.

Reply:
Then maybe rename it to .conf.example?